File: Beecrowd/1 - Beginner/1000 - 1099/1036 - Bhaskara's Formula/C++/1036_BhaskarasFormula.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    double a, b, c;
    cin >> a >> b >> c;

    double delta = b * b - 4 * a * c;

    if (delta < 0 || a == 0) cout << "Impossivel calcular" << endl;
    else {
        double r1 = (-b + sqrt(delta)) / (2 * a);
        double r2 = (-b - sqrt(delta)) / (2 * a);
        cout << "R1 = " << fixed << setprecision(5) << r1 << endl;
        cout << "R2 = " << fixed << setprecision(5) << r2 << endl;
    }
    
    return 0;
}

File: Beecrowd/1 - Beginner/1000 - 1099/1036 - Bhaskara's Formula/Python/1036_BhaskarasFormula.py
import math

a, b, c = map(float, input().split())

delta = b * b - 4 * a * c

if delta < 0 or a == 0:
    print("Impossivel calcular")
else:
    r1 = (-b + math.sqrt(delta)) / (2 * a)
    r2 = (-b - math.sqrt(delta)) / (2 * a)
    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")